modules/aws/route53/example/example.tf

```diff
@@ -0,0 +1,30 @@
+module "example_com" {
+  source = "../."
+
+  domain  = "example.com"
+  aliases = [
+    "example.org"
+  ]
+  alias_records = {
+    "example.org" = {
+      foo = {
+        name    = "foo"
+        type    = "CNAME"
+        records = ["www.example.com"]
+      }
+      something_interesting = {
```

Comment:
I may be being picky, but I would say put the TXT example on the canonical `records` rather than in the alias, and a lot of TXT records tend to be applied to the apex domain, but you could give an example like `google-site-verification` for instance (see answerdigital.com)

Reply:
See latest commit